opt/Makefile

```diff
@@ -222,9 +229,12 @@ endif
 pack: $(INSTALLED_TARGET)
 	$(SHOW)find $(INSTALL_DIR) -name "*.so" -exec chmod +x {} \;
 	$(SHOW)mkdir -p $(ROOT)/bin/artifacts
+ifneq ($(LITE), 1)
```

Comment:
REDISAI_LITE again. If we stick to this, it makes grepping easier in the future -especially having gone through the tflite code.